MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift

```diff
@@ -85,7 +78,8 @@ extension ActionToolbarContainer {
             button.setTitle("", for: .normal)
             button.setTitleColor(.secondaryLabel, for: .normal)
             button.expandEdgeInsets = UIEdgeInsets(top: -10, left: -10, bottom: -10, right: -10)
-            button.setInsets(forContentPadding: .zero, imageTitlePadding: style.buttonTitleImagePadding)
+            button.setInsets(forContentPadding: .zero, imageTitlePadding: 4)
+            button.showsLargeContentViewer = true
```

Comment:
The `showsLargeContentViewer` seems to be a workaround, right?

What I would have expected from this issue/PR is Dynamic Type-support for the four buttons (reply, reblog, favorite, share). Not that `UILargeContentViewerInteraction` is a bad thing, but it's only available for the huge accessibility Dynamic Type-values.

The font itself doesn't change: 

```swift
button.titleLabel?.font = .monospacedDigitSystemFont(ofSize: 12, weight: .regular)
```

Reply:
Correct, as I said in the OP the images cannot be easily resized in UIKit, but I could just resize the text.

Reply:
Update: figured out how to get the images to resize, but having some fun getting the icons to break onto 2 lines for larger text sizes while still retaining the current look on smaller text sizes